### ISSUE TITLE
Static Images with Metadata

### ISSUE THREAD

**Issue by ezrajrice** [tethysplatform/tethys#630]
Is there a reason to keep the metadata on the static files (PDF, JPG/JPEG, TIFF, GIF, PNG, PSD, BMP, RAW, CR2, CRW, PICT, XMP, DNG)?

These are getting flagged in penetration testing as "Metadata Information Leakage" with a CVSS Score of 2.6 (low risk). I've looked at the metadata and found that if we just make sure that we remove the metadata formats such as EXIF, IPTC, and XMP, then this shouldn't get flagged by most penetration testing software.

Example finding:
Request URL http://localhost:8000/static/tethys_portal/images/tethys-logo-25.png
Metadata found:
```json
{
  "PNG-IHDR": {
    "Image Width": "25",
    "Image Height": "25",
    "Bits Per Sample": "8",
    "Color Type": "True Color with Alpha",
    "Compression Type": "Deflate",
    "Filter Method": "Adaptive",
    "Interlace Method": "No Interlace"
  },
  "PNG-pHYs": {
    "Pixels Per Unit X": "11811",
    "Pixels Per Unit Y": "11811",
    "Unit Specifier": "Metres"
  },
  "XMP": {
    "XMP Value Count": "28",
    "Create Date": "2015-02-05T23:38:28-07:00",
    "Creator Tool": "Adobe Photoshop CC 2014 (Macintosh)",
    "Metadata Date": "2015-03-13T16:19:16-06:00",
    "Modify Date": "2015-03-13T16:19:16-06:00"
  },
  "PNG Chromaticities": {
    "White Point X": "31269",
    "White Point Y": "32899",
    "Red X": "63999",
    "Red Y": "33001",
    "Green X": "30000",
    "Green Y": "60000",
    "Blue X": "15000",
    "Blue Y": "5999"
  }
}
```

**Comment by swainn** [comment]
@ezrajrice I was actually unaware of the metadata on the images. I'm guessing it would not be a problem to remove that. Is there a good tool that can do this in batches?

**Comment by ezrajrice** [comment]
I've used [exiftool](https://exiftool.org/) on Ubuntu. It's supported on Windows, MacOS, and various flavors of Linux. I just found that the Linux install was smoother to install and use.

Installation on Ubuntu, Debian, Mint, Kali:  

> `$ sudo apt install libimage-exiftool-perl`

Installation on Fedora, CentOS, RedHat:  

> `$ sudo dnf install perl-Image-ExifTool.noarch`

List Metadata of given Image/File:

> `$ exiftool myimage.png`

Alternatively you can list specific metadata by piping through grep

> `$ exiftool myimage.png | grep "XMP"`

To remove metadata from single file:

> `$ exiftool -all= myimage.png`
> **Note:**
> This makes a copy of the files with the originals being prepended with "_original"

Remove metadata from multiple files based on extension (within current directory):

> `$ exiftool -all= *.png`
> **Note:**
> This makes a copy of the files with the originals being prepended with "_original"

To remove metadata without keeping a copy of the original file use the `-overwrite_original` flag

> `$ exiftool -all= -overwrite_original *.png`

**Comment by swainn** [comment]
Addressed with #648
